.vscode/extensions.json

```diff
@@ -2,5 +2,9 @@
 
 {
     "recommendations": [
+        "streetsidesoftware.code-spell-checker",
+        "donjayamanne.githistory",
```

Comment:
Some of these are extremely opinionated, which is fine, but what was the thought behind some?

Reply:
Why are we committing this at all?

Reply:
So the way how it works, that if you open the project for the first time after cloning, it will popup a "suggested extensions" dialog where you can (if you want) install the extensions from the list. I can remove it if you think this would cause confusion. But yes, these are always going to be opinioned. Same as the upcoming discussion about .editorconfig ;)

Reply:
So removed this file for now :)